docs/application/web/get-started/overview.md

```diff
@@ -34,3 +34,17 @@ HTML5, CSS, and JavaScript:
 The first application examples show a simple application with a basic UI
```

Comment:
Please add your changed line 34.

-   [Web Addon](addon/first-addon.md)
    An addon is a software program that runs over web applications commonly.

Reply:
1. Do you want me to use 'Web Addon' instead of 'Addon' ? Is there any reason for that ? Like duplication of the legacy ?
2. I'll decide to follow your comment after completing to discussion on your another comment for line 38.

Reply:
I think the current the title of heading 2 (## Richness on your Web applications) is good.
By the way, 

---
I don't understand what "Like duplication of the legacy" means.  Legacy of  What???
Do you mean "Web application" is duplicated?

Anyway....
docs/application/web/get-started/overview.md:
The bullet item (Addon)  is not unclear. If I read the overview section,  'Addons' means 'Web application addons'.

docs/application/web/guides/applications/addon.md:
Heading 1(# Addon ) is the page title. When addon.md is displayed on the web site, developers will read the page tile first. 
So... it is better to write clear title.

**So... how about "Web application addons"?**

Reply:
@journey2w ,

- meaning of 'duplication of legacy'
    I wonder if you recommended to change the title 'Addon' to 'Web Addon' because the word 'Addon' is already used in other page of Tizen documents.

- changing the word 'Addon' to 'Web application addons'
    I agree that it's possible that readers may not think 'addon' is not related to web application when they only read or hear the word itself without any explanation. However, the name you suggested, 'Web application addons' seems to be long a little bit. How about to use 'Web addons' like your first suggestion ?

Reply:
One another considering point is, that we have several other links to each "Create Your First ..." pages in the overview.md.
As you can see, each URLs are linked to "_something_ Web _something_  application" strings.

-   Mobile Web application
    A mobile application is ...
-   Wearable Web application
    A wearable application is ...
-   Wearable Web watch	application
    A wearable watch application is ...
-   Wearable Web widget application
    A wearable widget application is ...
-   TV Web application
    A TV application is ...

So the "Web application addons" is following the similar format with other list items.
I agree with that "Web application addons" seems a little bit long, but how about using the similar format with the existing list?

**Note** : The request to change to "Web application addons" is targeting two places.
   - The list menu in get-started/overview.md
   - The page title of guides/applications/addon.md

Reply:
@mijin85cho ,
I discussed internally on changing the item and title you suggested.
Like you know, it seems to be hard to change every 'addon' to 'Web application addon' in the context. So, how about changing the only followings ?:

- Navigation Menu
    - Web Application > Get Started > Create Addon  ==> **Create Web Application Addon**
    - Web Application > Guides > Applications > Addon ==> **Web Application Addon**
- Pages
    - application/web/get-started/overview.md
        - (bullet) Addon ==> (bullet) **Web Application Addon**
    - application/web/get-started/addon/first-addon.md
        - (Title) Create Your First Tizen Addon ==> **Create Your First Tizen Web Application Addon**
        - (Firtst Sentence) Welcome to Tizen Addon development! ==> **Welcome to Tizen Web Application Addon development!**
    - application/web/guides/applications/addon.md
        - (Title) Addon ==> **Web Application Addon**

Please verify.

Reply:
@hyundukkim I think your suggestion is really good.
Actually it is more than we requested. As we only have focused on 2 strings, you pointed out the missed points by us. Thanks for that.

Reply:
Thanks for verifying it.
Done